spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/liquibase/LiquibaseAutoConfiguration.java

```diff
@@ -125,6 +128,7 @@ public SpringLiquibase liquibase(ObjectProvider<DataSource> dataSource,
 			if (properties.getUiService() != null) {
 				liquibase.setUiService(UIServiceEnum.valueOf(properties.getUiService().name()));
 			}
+			liquibase.setCustomizer(customizer.getIfAvailable());
```

Comment:
This could be `customizer.ifAvailable(liquibase::setCustomizer)`. It would then only be set if the customizer's available which I think is slightly better than setting it to `null`.

Reply:
Done, thanks.